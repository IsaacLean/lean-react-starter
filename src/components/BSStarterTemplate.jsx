import React from 'react';

import Navbar from './Navbar';

const BSStarterTemplate = () => (
  <div id="bs-starter-template">
    <Navbar brand="Navbar" navItems={
      [
        {liClass: 'active', text: 'Home'},
        {text: 'Link'},
        {aClass: 'disabled', text: 'Disabled'}
      ]
    } />
    <div className="container">
      <div className="starter-template">
        <h1>Bootstrap starter template</h1>
        <p className="lead">
          Use this document as a way to quickly start any new project.
          <br />
          All you get is this text and a mostly barebones HTML document.
        </p>
      </div>
    </div>
  </div>
);

export default BSStarterTemplate;