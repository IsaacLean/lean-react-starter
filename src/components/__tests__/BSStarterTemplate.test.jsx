import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import React from 'react';

import BSStarterTemplate from '../BSStarterTemplate';

Enzyme.configure({ adapter: new Adapter() });

test('BSStarterTemplate renders correctly', () => {
  const component = Enzyme.shallow(<BSStarterTemplate />);
  expect(component).toMatchSnapshot();
});
