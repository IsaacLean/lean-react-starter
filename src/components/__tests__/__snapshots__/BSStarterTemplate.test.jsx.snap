// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BSStarterTemplate renders correctly 1`] = `
<div
  id="bs-starter-template"
>
  <Navbar
    brand="Navbar"
    navItems={
      Array [
        Object {
          "href": "/",
          "liClass": "active",
          "text": "Home",
        },
        Object {
          "href": "/",
          "text": "Link",
        },
        Object {
          "aClass": "disabled",
          "href": "#",
          "text": "Disabled",
        },
      ]
    }
  />
  <main
    className="container"
  >
    <div
      className="starter-template"
    >
      <h1>
        Bootstrap starter template
      </h1>
      <p
        className="lead"
      >
        Use this document as a way to quickly start any new project.
        <br />
        All you get is this text and a mostly barebones HTML document.
      </p>
    </div>
  </main>
</div>
`;
